src/shared/utils/index.js

```diff
@@ -256,13 +288,12 @@ const getLabelMaxWidth = (keys, displayFn) => keys.reduce((max, key) =>
  * @returns { number } - the width of the rendered text in pixels
  */
 export const getTextSize = (text, font = '12px noto sans') => {
-  const entities = new XmlEntities()
   let canvas = document.createElement('canvas')
   let context = canvas.getContext('2d')
   context.font = font
   let width = typeof text === 'number'
     ? context.measureText(text).width
-    : context.measureText(entities.decode(text)).width
+    : context.measureText(decodeXML(text)).width
```

Comment:
You know, I did try XmlEntities before using the html-entities package in Tamires and somehow it didn't work. I see it works now, maybe something was off when I tried. I was wondering you didn't pointed it out for my PR :).

Reply:
i didn't know about these particular implementations. the sole intent in this PR is to reduce the eventual package size, so I went functions with very sporadic usages and sought after native/in-house implementations to replace the packages